database/seeders/CustomerSeeder.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Console\Seeds\WithoutModelEvents;
+use Illuminate\Database\Seeder;
+use App\Models\Customer;
+use App\Models\UserMembership;
+use Illuminate\Support\Facades\Log;
+
+class CustomerSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     */
+    public function run(): void
+    {
+        if (Customer::exists()) {
+            Log::info('CustomerSeeder: Customer table is not empty. Skipping seeeding.');
+            return;
+        }
+        Log::info('CustomerSeeder: Seeding Customers and UserMemberships.');
+   
+        $superuser = Customer::factory()
```

Comment:
What's the point of the super user, out of curiosity. It just seems like they've been given all the training here so I'm wondering if that means it must be updated whenever we hardcode a new membership plan.

Reply:
The super user is a user that you can use to not have to worry about not having permissions for things. It's a good starting point to discovering/testing functionality without being restricted by lack of permissions. If you want to test/add functionality that it doesn't have, yes, you'll need to update it. Seed maintenance is a thing, but it shouldn't be a big lift. If someone wants to automate it further, that would certainly be welcome. This is just a starting point so that you don't have to go craft the first user in a blank database and figure out what all it needs in order to be functional. Seeds also serve as quick-start examples of what data affects a change in functionality.